src/components/Layouts/Authority.js

```diff
@@ -1,50 +1,66 @@
 import React, { Component } from "react";
 import { connect } from "react-redux";
-import { removeJob, raiseError, updateJob, removeError } from "../../lib/redux/actions";
+import { isRequest, raiseError } from "../../lib/redux/actions";
 
 class Authority extends Component {
   constructor(props) {
     super(props);
-   this.state = {
-    job: {
-      title:'',
-      category: '',
-      location:'' ,
-      img: ''
-    }
-   }
+    this.state = {};
     this.props.http.interceptors.request.use(
       async config => {
-        this.props.removeJob();
-        const job = this.state.job
-        this.props.updateJob(job);
-        this.props.removeError()
+        const is_request = true;
```

Comment:
Dư thừa cái này. nó không cần thiết. truyền thằng true or false vào function

EX: this.props.isRequest(true);